shared/tls/server/server.go

```diff
@@ -192,10 +202,10 @@ func defaultTLSHandler(conn net.Conn) {
 		defer c.Close()
 		var upstream net.Conn
 		conn.SetDeadline(time.Now().Add(5 * time.Second))
-		//TODO after we will have backends we can drop the hardcoded 443
+		//TODO after we will have backends we can drop the hardcoded 443.
```

Comment:
I'm surprised revive doesn't complain about the lack of space between `//` and `TODO`

Reply:
I do not think that spaces are mandatory, but if you feel strongly about them I can add them.